conwayste/src/ui/pane.rs

```diff
@@ -88,6 +90,19 @@ impl Pane {
         Ok(())
     }
 
+    pub fn get_widget_ids(&self) -> Vec<WidgetID> {
```

Comment:
It might be good to have a `recursive` bool here.

Reply:
Are you worried about stack usage? Not following how that would be better aside from that.

Reply:
No, more about the caller not always wanting the widget IDs recursively.

Reply:
Can we add it in if we see the use case later down the road?

Reply:
Sure, ok.